app/build.gradle

```diff
@@ -207,4 +207,12 @@ dependencies {
 
 }
 
+allprojects {
```

Comment:
We only have one project at the moment, but I imagine that we would want to apply these checks globally.